std/experimental/allocator/package.d

```diff
@@ -2221,13 +2221,13 @@ if (isInputRange!R)
             toFill.uninitializedFillDefault;
         }
 
-        for (; !range.empty; range.popFront, toFill.popFront)
+        for (; !range.empty; range.popFront, toFill = toFill[1 .. $])
```

Comment:
`toFill.popFront` should work, no?

Reply:
[`popFront` does some decoding of its own](https://github.com/dlang/phobos/blob/a394c1348b4b73765bd0bbf4d08b0a8548c5778b/std/range/primitives.d#L2257), which could cause some uninitialised chars and wchars to be skipped